mozilla-mobile/android-components/components/compose/tabstray/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Message announced to the user when tab tray is selected with 1 tab -->
+    <string name="mozac_tab_counter_open_tab_tray_single">1 open tab. Tap to switch tabs.</string>
```

Comment:
@pocmo Sorry, I just realized something else. For this string, can you really "Tap to switch tabs" if there is only one open tab? Sorry if I'm misunderstanding what actually is supposed to happen here :)

Reply:
I didn't really create this string. I took it from an older implementation and this string already exists in an other file:
* https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-mobile/android-components/components/ui/tabcounter/src/main/res/values/strings.xml#L6

This is the content description (read by screen readers) on the tab counter button, that, when pressed, opens the tab tray.